app/src/main/java/com/example/konrad/decentwork/First_Look.java

```diff
@@ -0,0 +1,31 @@
+package com.example.konrad.decentwork;
+
```

Comment:
Isn't it better to make splashscreen like thishttps://android.jlelse.eu/right-way-to-create-splash-screen-on-android-e7f1709ba154 and instantly go to the login without using and button?